social-network/src/articles-yours.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+
+export default class YourArticle extends Component {
+    /* I recieve the following warning on this .js file as well as articles-missed.js and articles-view.js:
+    *
+    * 'Useless constructor  no-useless-constructor'
+    * 
+    * Am i not utilizing my constructor functon correctly?
+    */
+
+

```

Comment:
You are getting this issue because you have a constructor function but are not doing anything with it. If you had to initialize some parameters in State or bind the 'this' for some functions etc you will do that in a React Components constructor. Since you are not doing that here you can remove the lines 12-14 to avoid those warnings.